styles/style.css

```diff
@@ -6,3 +6,131 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  margin: 0;
+  color: 1a1a1a;
+  font-family: "Poppins";
+}
+h3 {
+  color: #fff;
+}
+
+.navbar {
+  display: flex;
+  position: fixed;
+  width: 100%;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.spotify-logo {
+  width: 150px;
+  padding: 15px;
+}
+
+.navigation ul {
+  display: flex;
+  justify-content: space-between;
+}
+
+.navigation ul li {
+  list-style: none;
+  padding: 0 10px;
+  font-weight: 400;
+}
+
+h1 {
+  text-align: center;
+  font-size: 3.5rem;
+  margin-top: 0;
+  letter-spacing: 1px;
+}
+.landing {
+  align-content: center;
+  background-image: url(../images/landing.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 600px;
+}
+.landingText {
+  align-content: center;
+  color: white;
+  padding-top: 200px;
+  text-align: center;
+}
+
+.landing p {
+  font-size: 20px;
+  font-weight: 0;
+  letter-spacing: 1px;
+}
+.service {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
I would personally have used space-around here since it currently pushed all the way to to sides, making the design look a bit off:)
You can also fix the issue with setting a width of the container